dashboard/config/scripts/netsim.script

```diff
@@ -14,7 +14,7 @@ level 'NetSim_ManualDNS'
 level 'NetSim_AutomaticDNS'
 
 stage 'Internet Simulator Templates'
-level 'blockly:NetSim:variant1'
-level 'blockly:NetSim:variant2'
-level 'blockly:NetSim:variant3'
+level 'NetSim_Variant1'
+level 'NetSim_Variant2'
+level 'NetSim_Variant3'
```

Comment:
Not to my knowledge; I didn't identify it as a bug, I just figured I was using the system incorrectly.  We're trying to totally retire the `levels.js` files anyway, right?